features/issues/components/issue-list/issue-row.tsx

```diff
@@ -30,20 +30,26 @@ export function IssueRow({ projectLanguage, issue }: IssueRowProps) {
           alt={projectLanguage}
         />
         <div>
-          <div className={styles.errorTypeAndMessage}>
-            <span className={styles.errorType}>{name}:&nbsp;</span>
+          <div className={styles.errorTypeAndMessage} data-cy="issues-message">
```

Comment:
I had a look around and it seems that there are two opinions about test ids: developers often don't like them (e.g. Kent C Dodds recommends not to use them if possible in the [Testing Library docs](https://testing-library.com/docs/queries/about/#priority)) while many QA engineers favor them because they allow for test stability (texts can change often and accessing elements by text makes the tests break frequently). I think the latter has a good point